ONWServices.Tests/Unit/NewGameTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ONWServices.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONWServices.Tests.Unit
+{
+    //Game statuses: New, Available, Night Phase, Information Review, Discussion, Round Summary, (Back to Night Phase), Game Summary, Closed
+    //Game must have a minimum of 3 players    
+    //Game can only have 1 Game Master
+
+    [TestClass]
```

Comment:
Non-blocking: I've actually been thinking about trying out `NUnit` instead of `MSTest`, since `NUnit` tests can be executed on the command line, as part of the deployment pipeline for instance.

I think `MSTest` is fine for now since we both know it but later on I'd like to give `NUnit` a try.